src/app/app.module.ts

```diff
@@ -4,61 +4,53 @@ import { FormsModule } from '@angular/forms';
 import { RouterModule, Routes } from '@angular/router';
 
 import { AppComponent } from './app.component';
-import { DashboardCMP } from './dashboard/dashboard.cmp';
-import { StreamCMP } from './stream/stream.cmp';
-import { Stream4WayCMP } from './stream-4way/stream-4way.cmp';
-import { ControlsCMP } from './controls/controls.cmp';
-import { GroupsCMP } from './groups/groups.cmp';
-import { SplashCMP } from './splash/splash.cmp';
-import { SplashReplayCMP } from './splash-replay/splash-replay.cmp';
-import { BracketCMP } from './bracket/bracket.cmp';
-import { StatsCMP } from './stats/stats.cmp';
-import { CreditsCMP } from './credits/credits.cmp';
-import { SwissCMP } from './swiss/swiss.cmp';
-import { IntermissionCMP } from './intermission/intermission.cmp';
-import { DivisionComponent } from './division/division.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
+import { StreamComponent } from './stream/stream.component';
+import { ControlsComponent } from './controls/controls.component';
+import { GroupsComponent } from './groups/groups.component';
+import { SplashComponent } from './splash/splash.component';
+import { BracketComponent } from './bracket/bracket.component';
+import { StatsComponent } from './stats/stats.component';
+import { CreditsComponent } from './credits/credits.component';
+import { IntermissionComponent } from './intermission/intermission.component';
+import { CommentatorsComponent } from './commentators/commentators.component';
```

Comment:
Thank you, I've been meaning to rename these forever. Like I think I actually yelled at Terra about it more than once lol.